cmd/bbolt/README.md

```diff
@@ -72,7 +72,7 @@
   ```bash
   $bbolt version
   bbolt version: 1.3.7
-  Go Version: go1.21.5
+  Go Version: go1.21.6
```

Comment:
minor comment: It's just an example, we don't need to update it each time when we bump go version.